frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_file_exporter_widget.dart

```diff
@@ -248,18 +246,17 @@ class _AppFlowyFileExporter {
   ) async {
     final failedFileNames = <String>[];
     final Map<String, int> names = {};
-    final documentService = DocumentService();
     for (final view in views) {
       String? content;
       String? fileExtension;
       switch (view.layout) {
         case ViewLayoutPB.Document:
-          final document = await documentService.openDocument(view: view);
-          document.fold((l) => Log.error(l), (r) {
-            final json = r.toDocument()?.toJson();
-            if (json != null) {
-              content = jsonEncode(json);
-            }
+          final documentExporter = DocumentExporter(view);
+          final result = await documentExporter.export(
+            DocumentExportType.json,
+          );
+          result.fold((l) => Log.error(l), (json) {
+            content = json;
           });
```

Comment:
Comma for formatting

Reply:
It is acceptable as long as the CI doesn't raise any issues.